src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormatter.cs

```diff
@@ -179,12 +179,16 @@ private TextEdit[] MapEditsToHostDocument(RazorCodeDocument codeDocument, TextEd
 
             // Assuming the C# formatter did the right thing, let's extract the indentation offset from
             // the line containing trivia and token that has our attached annotations.
-            ExtractTriviaAnnotations();
-            ExtractTokenAnnotations();
+            ExtractTriviaAnnotations(context, formattedRoot, formattedText, desiredIndentationMap);
```

Comment:
This was kind of a random change, but I was initially debugging through the code and was initially confused by what was going on here. I think it would be clearer as a static local function with the params explicitly passed in, but welcome to hear other opinions.

Reply:
Love it!